src/Appwrite/Platform/Workers/UsageHook.php

```diff
@@ -23,83 +24,71 @@ public function __construct()
         $this
             ->setType(Action::TYPE_WORKER_START)
             ->inject('register')
-            ->inject('cache')
-            ->inject('pools')
-            ->callback(function ($register, $cache, $pools) {
-                $this->action($register, $cache, $pools);
+            ->inject('getProjectDB')
+            ->callback(function ($register, callable $getProjectDB) {
+                $this->action($register, $getProjectDB);
             })
         ;
     }
 
     /**
      * @param $register
-     * @param $cache
-     * @param $pools
+     * @param $getProjectDB
      * @return void
      */
-    public function action($register, $cache, $pools): void
+    public function action($register, $getProjectDB): void
     {
-        Timer::tick(30000, function () use ($register, $cache, $pools) {
 
-            $offset = count(self::$stats);
-            $projects = array_slice(self::$stats, 0, $offset, true);
-            array_splice(self::$stats, 0, $offset);
-
-            foreach ($projects as $projectInternalId => $project) {
-                try {
-                    $dbForProject = new Database(
-                        $pools
-                            ->get($project['database'])
-                            ->pop()
-                            ->getResource(),
-                        $cache
-                    );
-
-                    $dbForProject->setNamespace('_' . $projectInternalId);
-
-                    foreach ($project['keys'] ?? [] as $key => $value) {
-                        if ($value == 0) {
-                            continue;
-                        }
+            $interval = (int) App::getEnv('_APP_USAGE_AGGREGATION_INTERVAL', '50000');
```

Comment:
This was 30s before, and is now 5s. Just making sure you are aware.

Reply:
its 50 sec no?